src/cargo/core/compiler/custom_build.rs

```diff
@@ -160,8 +160,7 @@ fn build_work<'a, 'cfg>(cx: &mut Context<'a, 'cfg>, unit: &Unit<'a>) -> CargoRes
                 Kind::Host => &bcx.host_triple(),
                 Kind::Target => bcx.target_triple(),
             },
-        )
-        .env("DEBUG", debug.to_string())
+        ).env("DEBUG", debug.to_string())
```

Comment:
I think these formatting changes are likely a bug in the latest rustfmt (https://github.com/rust-lang-nursery/rustfmt/issues/2932).

Reply:
Thanks for the link, @ehuss! Good to know it's classified as a bug 😄